frontend/src/app/components/block/block.component.ts

```diff
@@ -269,7 +274,7 @@ export class BlockComponent implements OnInit, OnDestroy {
       this.isLoadingOverview = false;
     });
 
-    if (!this.indexingAvailable) {
+    if (!this.indexingAvailable || !this.stateService.env.AUDIT) {
```

Comment:
@mononaut is this conditional correct? Does it make sense to have audit enabled but indexing disabled?

Reply:
it's correct, but maybe overly verbose - you're right that `AUDIT` should imply `indexingAvailable`